data/news/platform/platform-2023-10.md

```diff
@@ -0,0 +1,115 @@
+
+**What:** Releasing opam 2.2 with native support for Windows and making the official `opam-repository` usable on Windows.
+
+**Activities:**
+- Not much to report on this month. More to come for the November update (finalisation of `setenv` and release of opam 2.2 alpha3).
+
+## **[`odoc`]** Add Search Capabilities to `odoc` ([W25](https://ocaml.org/docs/platform-roadmap#w25-generate-documentation))
+
+**Contributed by:** @panglesd (Tarides), @EmileTrotignon (Tarides), @julow (Tarides), @jonludlam (Tarides)
+
+**Why:** The in-package search added in OCaml.org's central package documentation has been very well received by the community and improves how users navigate and discover OCaml documentation. We're upstreaming it to `odoc` to bring it into the local documentation as well and provide more advanced features, like searching by type.
+
+**What:** We're adding support in `odoc` for pluging in a search engine! `odoc` provides the UI (a search bar) and will generate a search index (that can also be used to be integrated into other search engine like Elasticsearch). We're also building a default client-side search engine based on Sherlodoc.
+
+**Activities:**
+- We've merged the PR adding [new search capabilities](https://github.com/ocaml/odoc/pull/972)! :tada: This will ship in the upcoming version of `odoc` 2.4.0.
+- We are currently experimenting with building an `odoc` search engine based on [Sherlocode](https://github.com/art-w/sherlodoc) --[art-w/sherlodoc#4](https://github.com/art-w/sherlodoc/pull/4).
+
+## **[`odoc`]** Syntax for Images and Assets in `odoc` ([W25](https://ocaml.org/docs/platform-roadmap#w25-generate-documentation))
+
+**Contributed by:** @panglesd (Tarides), @jonludlam (Tarides), @dbuenzli, @gpetiot (Tarides)
+
+**Why:** Allow package authors to write rich documentation, making it more useful and improving OCaml developer experience by providing an ecosystem of high-quality documentation for OCaml packages.
+
+**What:** We're adding support for assets and new syntax to embed medias (images, audio, and videos).
+
+**Activities:**
+- We've decided to rework the asset support design a little to bring it more in line with how other elements are handled, such as modules, types, and values. The intent is to ensure the code is straightforward to maintain in the long term. -- [ocaml/odoc#1002](https://github.com/ocaml/odoc/pull/1002)
+- We continued reviewing the PR adding syntactic support for media. There were some good discussions relating to missing assets, and it looks to be in good shape to go in once the assets PR is merged. -- [ocaml/odoc#1005](https://github.com/ocaml/odoc/pull/1005)
```

Comment:
```suggestion
- We continued reviewing the PR, adding syntactic support for media. There were some good discussions relating to missing assets, and it's ready to go in once the assets PR is merged. -- [ocaml/odoc#1005](https://github.com/ocaml/odoc/pull/1005)
```

Even if you don't like the rewrite or it's just inaccurate, the comma must go before "adding" in the first sentence.